test/integration/mentor_configure_test.rb

```diff
@@ -14,8 +14,26 @@ class MentorConfigureTest < ActionDispatch::IntegrationTest
     get mentor_configure_path
 
     assert_select "form" do
-      assert_select "input[name=?][checked=checked]", "track_id[#{track_one.id}]"
-      assert_select "input[name=?]", "track_id[#{track_two.id}]"
+      assert_track_selected(track_one.id)
+      assert_track_not_selected(track_two.id)
     end
+
+    # User unclicks their track and cliks the other path
+    # effectively switching which tracks they belong to
+    put mentor_configure_path("track_id" => { track_two.id.to_s => true })
```

Comment:
What do you think about also adding a system test to test the form works?

Something like (this won't work but to give you clarity on what I mean):
```
get mentor_configure_path
click("input[name=track_id[#{track.id}]]")
click_button("Continue")
assert_equal [track], user.mentored_tracks 
```